### ISSUE TITLE
Trouble NetBooting non T2 USB-C Macs

### ISSUE THREAD

**Issue by putnajoe** [jamf/NetSUS#130]
I have downloaded versions 4.2.1 and 5.0 of the NetSUS appliance, created new NetBoot/NetInstall images using AutoCasperNBI and AutoNBI and uploaded them to the appliances that are running on the same ESXi 6.5 U2 host. Under version 4.2.1, everything works as expected and I can NetBoot all Macs, however, on version 5 I am unable to NetBoot Macs that have USB-C. I am getting the following errors when booting in verbose mode:

ERROR!!! Load prelinked kernel with status 0x8000000000000002
Error loading kernel cache (0x2)

I have even tried version 5.0.1 from the Mondada Github page on a Ubuntu 16.04 LTS vm and get the same error. 

I have verified that the TFTP block size is set to 1460 in all instances and I am only running one appliance at a time on the ESXi host.

In summary, I am confused why the same image works on version 4.2.1 and not on version 5.0.

**Comment by egenethebest** [comment]
Hello, I can confirm the exact same issue on NetSus v5.0.1
I can successfully boot my NetRestore and NetInstall images on older macs but on 18,1 iMac (which is a non T2 USB-C Mac) I get:
ERROR!!! Load prelinked kernel with status 0x8000000000000002
Error loading kernel cache (0x2)

I also verified that my TFTP block size is set to 1460, and we only have one (v5.0.1) appliance running.
